### PR TITLE
[AdminListBundle] avoid translations freakout when the key is true or false without escaping

### PR DESCRIPTION
**Pull request by Numkil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #1899

Translations would freak out and display 1 or '' in english locale without adding these quotes.

### CODE REVIEW

File: src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -14,8 +14,8 @@ kuma_admin_list:
 
 # Filters
 filter:
-    true: true
-    false: false
+    "true": true
```
Comment (acrobat) [inline src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml]: shouldn't the value be quoted instead of the key?

Comment (acrobat) [reply src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml]: I think both the key and the value need to be quoted, so it seems when I'm testing..

Comment (Numkil) [reply src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml]: For me it worked if i only commented the key.

Comment (Numkil) [reply src/Kunstmaan/AdminListBundle/Resources/translations/messages.en.yml]: but i will update everything to be sure

**Pull request merged**